exercises/grains/grains.py

```diff
@@ -1,6 +1,6 @@
-def on_square():
+def on_square(integer):
```

Comment:
`integer_number` sounds more natural, don't you agree?

Reply:
I agree - though I would say having something mention that the integer_number is the square would make this a bit more obvious.